R-package/recreate-configure.sh

```diff
@@ -4,6 +4,14 @@
 # this script should run on Ubuntu 18.04
 AUTOCONF_VERSION=$(cat R-package/AUTOCONF_UBUNTU_VERSION)
 
+# R packages cannot have versions like 3.0.0rc1, but
+# 3.0.0-1 is acceptable
+LGB_VERSION=$(cat VERSION.txt | sed "s/rc/-/g")
```

Comment:
I'm confused a little bit. Where do we copy `VERSION.txt` from the root repo folder to `R-package`? Or how is it working?

Reply:
you don't need to! When you run `./R-package/recreate-configure.sh`, the effective working directory is the root of the repo. So no need to copy it from anywhere.

Reply:
Aha, got it! Thanks for the explanation!